### PR TITLE
feat(agenda): add navbar in agenda, closes #448

### LINKED ISSUE #448: Add tab view in agenda
- Day 1 - Alpha Track
- Day 1 - Beta Track
- Day 2 - Alpha Track
- Day 2 - Beta Track

### PR DESCRIPTION
**Pull request by TheJoin95**
Closes #448 

image
image

@nicpuppa I did not add the speaker at the schedule, we can do it in other issues

### CODE REVIEW

**Comment by TheJoin95** [comment]
There are some speakers to be added in the translations, espcially on the last day on beta track. I think it's better to handle this in a separate issue

File: pages/agenda.tsx [hunk 1]
```diff
@@ -26,39 +39,102 @@ export default function Agenda() {
...
+              <AgendaCard hour='14:05' isBreak={false} speakerName={s('valeria.name')} talkTitle={s('valeria.talk_title')} icon='/icons/microphone.svg' position='left' link={s('valeria.id')}/>
+              <AgendaCard hour='14:45' isBreak={false} speakerName={s('santosh.name')} talkTitle={s('santosh.talk_title')} icon='/icons/microphone.svg' position='right' link={s('santosh.id')}/>
+              <AgendaCard hour='15:30' isBreak={false} speakerName={s('federico.name')} talkTitle={s('federico.talk_title')} icon='/icons/microphone.svg' position='left' link={s('federico.id')}/>
+              <AgendaCard hour='16:15' isBreak={false} speakerName={s('sabrina.name')} talkTitle={s('sabrina.talk_title')} icon='/icons/microphone.svg' position='right' link={s('sabrina.id')}/>
+              {/* <AgendaCard hour='16:00' isBreak={true} breakTitle='Coffee break & Giveaway' icon='/icons/coffee.svg' position='right'/> */}
+              <AgendaCard hour='17:00' isBreak={false} speakerName={s('andrei.name')} talkTitle={s('andrei.talk_title')} icon='/icons/microphone.svg' position='left' link={s('andrei.id')}/>
```
Comment (nicpuppa) [inline pages/agenda.tsx]:
```suggestion
              <AgendaCard hour='17:00' isBreak={false} speakerName={s('andrey.name')} talkTitle={s('andrey.talk_title')} icon='/icons/microphone.svg' position='left' link={s('andrey.id')}/>
```

**Review by nicpuppa — CHANGES_REQUESTED** [review]
Overall good work, but we can use TBD for speaker not present yet. I will add them in the next speakers PR

File: pages/agenda.tsx (hunk 1 above)
Comment (TheJoin95) [reply pages/agenda.tsx]: I did remove the not confirmed yet speakers. This commit is missing the speakers translation part but no error it's throw. I believe we can merge this as it is or after you've added the missin translation for the already confirmed speaker

Comment (nicpuppa) [reply pages/agenda.tsx]: I don't like that tbh

**Comment by TheJoin95** [comment]
@404answernotfound @davideimola @Wabri are you able to add in this PR or in a following one the missing speakers ?

**Comment by TheJoin95** [comment]
@nicpuppa should work now

**Review by nicpuppa — APPROVED** [review]

**Pull request merged**